packages/commonwealth/shared/types.ts

```diff
@@ -1,5 +1,5 @@
-import { ChainAttributes } from 'server/models/chain';
-import { ChainEventAttributes } from 'server/models/chain_event';
+import { ChainAttributes } from '../server/models/chain';
+import { ChainEventAttributes } from '../server/models/chain_event';
```

Comment:
Technically this constitutes a bad cross-import between client and server. I think if we wanted to be clearer about this, we could use `import type`, but ideally we can avoid doing this at all.